components/sponsors/ResponsiveImage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface ResponsiveImageProps {
+  src: string;
+  alt?: string;
+  className?: string;
+  dataTestId: string;
+}
+
+/*
+ * This component displays the responsive image of the sponsors.
+ * @param {ResponsiveImageProps} props - The props for ResponsiveImage component.
+ * @param {string} props.src - Source of the image displayed.
+ * @param {string} props.alt - Alternative text for the image, important for accessibility.
+ * @param {string} props.className - Additional CSS class names for styling the image.
+ * @param {string} props.dataTestId - A string for identifying the element during testing.
+ */
+const ResponsiveImage: React.FC<ResponsiveImageProps> = ({ src, alt, className, dataTestId }) => {
```

Comment:
Why to create separate component for Responsive images? We can add this property to component of respective category.

Reply:
> HDI logo size is way too huge as compared to others, so probably we have to built a component that restricts the logo size with fixed dimension.
I thought that creating a component was required. No worries, I will do it without a new component.